build/Dockerfile

```diff
@@ -1,7 +1,13 @@
-FROM alpine:3.8
+FROM openshift/origin-release:golang-1.10 AS build
+ENV OPERATOR_PATH=/go/src/github.com/openshift/dedicated-admin-operator
+COPY . ${OPERATOR_PATH}
+WORKDIR ${OPERATOR_PATH}
+RUN make build
```

Comment:
How about `RUN yum update -y && yum clean all && make build` ?

Reply:
Guess would exclude go in the update.  Unless we don't care about the version of go.

Reply:
done, excluding the golang package from the yum update